internal/pkg/template/templates/workloads/services/backend/cf.yml

```diff
@@ -74,7 +97,20 @@ Resources:
             Action:
               - sts:AssumeRole
       Path: /
+      ManagedPolicyArns:
+        - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
       Policies:
+      {{- if .ALBEnabled}}
+        - PolicyName: "DNSandACMAccess"
```

Comment:
Should we give it a better name?

Reply:
yeah, it's not super representative - it's the same policy name we give the LBWS, so let's leave it to be consistent? long term, we should probably get rid of the custom resource role and just have a single role per lambda

Reply:
It's just we don't have ACM related actions here :think

Reply:
yeah....................................................good point haha. i'll change it for the backend service for now

Reply:
lmk if you don't like the name :)